tests/send-to-helix.proj

```diff
@@ -0,0 +1,101 @@
+<Project Sdk="Microsoft.DotNet.Helix.Sdk" InitialTargets="BuildHelixWorkItems" DefaultTargets="Test">
+
+  <PropertyGroup>
+    <Language>msbuild</Language>
+
+    <_workItemTimeout>00:20:00</_workItemTimeout>
```

Comment:
What's the difference between this and the `TestSessionTimeout` in the `.runsettings` file? Why are they 2 different values?

Reply:
The timeout defined in the yml is for the CI *step*, so the whole step that sends items to helix and waits on them. It would include the time taken while waiting for agents to be assigned, and the time take to run the tests on helix.

The timeout here is only for the time taken on helix, per helix item.

The timeout for the step is larger mainly to account for helix wait times, which can vary depending on how busy the queue is.

Reply:
What is the timeout in the `.runsettings` file for?

Reply:
> What is the timeout in the `.runsettings` file for?

IIUC, that is a timeout for individual tests.

Also, it should have been 5minutes but I had bumped it up when I was getting timeouts and forgot to revert that.